library/SpamFilter/PanelSupport/Cpanel.php

```diff
@@ -1953,7 +1953,7 @@ public function getDomainUser($domain)
 
             if (is_array($domains)) {
                 foreach ($domains as $d) {
-                    if ($d['domain'] == $domain && isset($d['owner_domain'])) {
+                    if ($d['domain'] !== $domain && isset($d['owner_domain'])) {
```

Comment:
This seems strange. Did you check that this works properly and doesn't affect anything for actual remote domains?